src/WindowsTracing/ClockUtils.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_TRACING_CLOCK_UTILS_H_
+#define WINDOWS_TRACING_CLOCK_UTILS_H_
+
+#include <profileapi.h>
+
+#include <cstdint>
+
+namespace orbit_windows_tracing::clock_utils {
```

Comment:
We agree not to use nested namespaces: https://github.com/google/orbit/blob/main/DEVELOPMENT.md#namespaces
In this case you can probably just remove the inner one.

Reply:
Done.